### PR TITLE
Wallentx/prerelease

### PR DESCRIPTION
**Pull request by wallentx**
The working state of code in this PR:
A `PR check` from `<dev branch>` to `main` - https://github.com/wallentx/billy-blockchain/pull/2/checks
https://github.com/wallentx/billy-blockchain/actions
https://github.com/wallentx/billy-blockchain/releases

### CODE REVIEW

File: .github/workflows/build-windows-installer.yml [hunk 1]
```diff
@@ -190,33 +220,27 @@ jobs:
...
-
     - name: Upload Installer to artifacts
       uses: actions/upload-artifact@v3
       with:
         name: chia-installers-windows-exe-intel
         path: ${{ github.workspace }}\chia-blockchain-gui\release-builds\
 
     - name: Install AWS CLI
-      if: steps.check_secrets.outputs.HAS_AWS_SECRET
+      if: env.FULL_RELEASE == 'true' && steps.check_secrets.outputs.HAS_AWS_SECRET
```
Comment (cmmarslender) [inline .github/workflows/build-windows-installer.yml:230]: This step should not just be on full release - we also need this to upload dev installers - should just be checking if HAS_AWS_SECRET like before

File: .github/workflows/build-windows-installer.yml [hunk 2]
```diff
@@ -190,33 +220,27 @@ jobs:
...
       run: |
           msiexec.exe /i https://awscli.amazonaws.com/AWSCLIV2.msi
 
     - name: Configure AWS Credentials
-      if: steps.check_secrets.outputs.HAS_AWS_SECRET
+      if: env.FULL_RELEASE == 'true' && steps.check_secrets.outputs.HAS_AWS_SECRET
```
Comment (cmmarslender) [inline .github/workflows/build-windows-installer.yml:235]: This step should not just be on full release - we also need this to upload dev installers - should just be checking if HAS_AWS_SECRET like before

**Review by cmmarslender — CHANGES_REQUESTED** [review]

File: .github/workflows/build-windows-installer.yml (hunk 1 above)
Comment (wallentx) [reply .github/workflows/build-windows-installer.yml]: fixed

File: .github/workflows/build-windows-installer.yml (hunk 2 above)
Comment (wallentx) [reply .github/workflows/build-windows-installer.yml]: fixed

**Comment by wallentx** [comment]
Dev build on Windows11

**Comment by wallentx** [comment]
Alpha build on Windows11
(Alpha is sort of a reserved version tag for validation of CI functionality)

**Comment by wallentx** [comment]
RC Build on Windows 11

**Comment by wallentx** [comment]
windows8 1
windows8 1-2
windows8 1-3
windows10-1
windows10-2
windows10-3

**Comment by wallentx** [comment]
server2012-dev3
server2012-a3
server2012-rc4
server2016-dev3
server2016-a3
server2016-rc4


server2022-dev3
server2022-a3
server2022-rc4

**Review by cmmarslender — APPROVED** [review]

**Review by wjblanke — APPROVED** [review]
aok

**Pull request merged**